src/bot/services/user_handling.py

```diff
@@ -0,0 +1,17 @@
+from pubsub import pub as pub
```

Comment:
```suggestion
from pubsub import pub
```
there is no need to do `import x as x` as this is giving it an alias that is the same as the name